utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -1456,7 +1456,10 @@ int main(int argc, char **argv)
 
         if (testType == 0)
         {
-            cout << "\n\nGPU Backend Wall Time: " << wallTime <<" ms/batch"<< endl;
+            cout <<"\n\n";
```

Comment:
please remove this and combine with the next cout statement

Reply:
Hi Rajy
The current implementation ensures a clear separation between different iterations and makes it easier to read the output. If we remove the "\n\n" and combine it with the next cout statement, it will introduce inconsistency in the output formatting. Specifically, it will result in either four new lines or just two lines, depending on the iteration count, which may lead to confusion. Keeping the \n\n helps maintain a clean and consistent format for the output.